src/Tizen.Applications.Common/Tizen.Applications/Application.cs

```diff
@@ -82,6 +82,47 @@ public ApplicationInfo ApplicationInfo
             }
         }
 
+        /// <summary>
+        /// Gets the current device orientation.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public DeviceOrientation CurrentDeviceOrientation
```

Comment:
I think It seem to valid only for UIApplication

Reply:
I removed this attribute. I'll move it to CoreUIApplication